pages/admin/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Linus_Aakerberg, Stian Kornbakk, @Menubrea" />
+    <meta name="team" content=" FE_Offer  " />
+    <link rel="stylesheet" href="/dist/css/custom/admin/index.css" />
+    <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js" defer></script>
+    <script src="/index.js" type="module"></script>
+    <script src="/src/js/index.js" type="module"></script>
+    <script type="module" src="/src/js/api/users/getAllUsers.js"></script>
+    <script src="/src/js/views/admin/showUsers.js" type="module"></script>
+    <script src="/src/js/views/admin/showListings.js" type="module"></script>
```

Comment:
Why link all of them here? Thats why we have the switch in the index file...

So i would like it to be in a new case like it is for the other pages.